recipe/conda_build_config.yaml

```diff
@@ -169,6 +169,8 @@ pin_run_as_build:
     max_pin: x
   flann:
     max_pin: x.x.x
+  fmt:
+    max_pin: x
```

Comment:
Can you add `run_exports` to the recipe and remove it from here?

Reply:
I opened a PR but it has not been merged yet.

Reply:
I can remove this line assuming that the PR will be merged soon. Is that what you had in mind?

Reply:
xref: https://github.com/conda-forge/fmt-feedstock/pull/11

Reply:
If I understand well, both this PR as well as https://github.com/conda-forge/fmt-feedstock/pull/11 should be merged to get proper pinning.